content/en/past-builds/mar-2021/m12.adoc

```diff
@@ -0,0 +1,203 @@
+---
+title: "How to build a PAPRa m12 Battery Controller (Mar 2021)"
+linkTitle: "PAPRa m12 Battery Controller (Mar 2021)"
+weight: 1
+toc_hide: true
+description: >
+  How-To guide for building the Tetra PAPRa m12 battery controller (March 2021 version).
+---
+
+== Quick Synopsis
+
+The Tetra Powered Air Purified Respirator requires power to function, and the m12 battery controller provides that power for these models.  The controller consists of internal circuitry, a potentiometer to adjust fan speeds, a cable to connect to the fan box, and a port for a Milwaulkee M12 Battery to be inserted.
+
+This build differs from the v1 version in that it uses a custom-built circuitboard to remove many of the difficult steps of the build, add a battery charge indicator, and a separate power port for use in a car.  If you cannot get link:../m12-circuit[the custom board], then please refer to the v1 version of this guide.
+
+*WARNING:* we provide no warranty, implicit or explicit, that this device will work up to an N95 specification, when built outside of a facility rated to do so.  The only way to truly test one of these devices using something like a https://tsi.com/products/respirator-fit-testers/portacount-respirator-fit-tester-8038/[Portacount] under controlled conditions.
+
+== Parts Required
+
+This build will require printed parts, purchased parts, and several tools.  Here is a picture of all the components used in this build:
+
+[link=https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-NWTC2Dz/0/39d6081d/5K/_MMR0198-5K.jpg]
+image::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-NWTC2Dz/0/39d6081d/L/_MMR0198-L.jpg[]
+
+== Assembly Diagram == 
+
+{{< embed-pdf hidePaginator=true url="/D-TET-001102-1 M12 PAPRA Controller Drawing.pdf" >}}
+
+## 3D Printed Parts
+
+These parts should be printed at 0.2 spacing:
+
+.Table Printed Parts
+|===
+| Part | Quantity | Material | Supports Required? | STL File Location
+
+| The Body
+| 1 
+| PLA
+| Yes
+| https://github.com/tetrabiodistributed/papr-print-files/blob/main/STLs/battery_controller/P-TET-000104-3%20M12%20Controller%20Body.stl
```

Comment:
print file should point to a release or commit, not `main`

Reply:
Good catch, fixed